docs/install_from_source.md

```diff
@@ -36,20 +36,25 @@ To install pgai from source on a PostgreSQL server:
 1. Clone the pgai repo at the latest tagged release:
 
     ```bash
-    git clone https://github.com/timescale/pgai.git --branch extension-0.6.0
+    git clone https://github.com/timescale/pgai.git --branch extension-0.8.0
     cd pgai
     ```
 
 1. Install the `pgai` PostgreSQL extension:
 
     ```bash
-    just ext install
+    sudo just ext install
     ```
+    
+    Note: The install requires sudo to create the following files (in system-owned paths):
```

Comment:
Should we add to the note explicitly mentioning that if you don't want to use sudo the user that's running the command needs write access to these folders?